quarry/web/worker.py

```diff
@@ -148,6 +148,13 @@ def run_query(query_run_id):
             celery_log.info("Stopped run for qrun:%s", qrun.id)
         else:
             write_error(qrun, e.args[1])
+    except pymysql.err.OperationalError as e:
+        qrun.status = QueryRun.STATUS_FAILED
+        conn.session.add(qrun)
+        conn.session.commit()
+        celery_log.error(
+            "OperationalError for qrun:%s, error: %s", qrun.id, str(e)
```

Comment:
I wonder if there are better ways to stringify errors than `str(e)`? May be something from the [traceback module](https://docs.python.org/3/library/traceback.html)?

Reply:
Added traceback

```
tb_str = "".join(
            traceback.format_exception(
                etype=type(e), value=e, tb=e.__traceback__
            )
        )
        celery_log.error(
            "OperationalError for qrun:%s, error: %s, traceback: %s",
            qrun.id,
            str(e),
            tb_str,
        )
```